common/types.go

```diff
@@ -264,7 +264,6 @@ type Association struct {
 	// AssociationType is the type of association.
 	AssociationType string         `json:"associationType,omitempty"`
 	Raw             map[string]any `json:"raw,omitempty"`
-	Fields          map[string]any `json:"fields,omitempty"`
```

Comment:
I added this in a previous PR before realizing that we don't support specifying fields for associated objects. We don't need this for now.